site/zenodo_rdm/legacy/requests/utils.py

```diff
@@ -82,3 +83,26 @@ def submit_record_upgrade_request(record, uow, comment=None):
     return current_requests_service.execute_action(
         system_identity, request_item._request.id, "submit", data=comment, uow=uow
     )
+
+
+@unit_of_work()
+def submit_community_manage_record_request(user_id, uow, comment=None):
```

Comment:
Request is created per each `user_id`. 
Question: the request should not be created for those users who don't have any records yet. Should this chech happen here or it will be applied only for needed users somehow during migration? ping @slint

Reply:
Users without records don't matter in this case, since there won't be any permission flags added to query against (i.e. the query will return no results).

Reply:
Yes that's true, but my point is, that the request will still submitted for those users (i didn't add any checks here because was not sure if it's done in migration, but if you plan to just apply this for all the users, i should add something)

Reply:
Ah ok, I understood now. 

So don't worry, when we run the script that calls this function to create the requests, we'll take that into account there, so no need to have extra checks in this code (just to keep it simpler).

Reply:
Ok, thank you for clarifying!